docs/source/gbd2017_models/concept_models/vivarium_sanofi_zenon/concept_model.rst

```diff
@@ -123,12 +123,12 @@ The probability of side effects is also taken from the literature, and if a pers
 BAU parameter data tables
 +++++++++++++++++++++++++
 
-.. csv-table:: Current prescriptions
-   :file: current_rx.csv
-   :widths: 30, 20, 10, 10
+.. csv-table:: Table 1: Probability of having LDL-c measured
```

Comment:
Patients are initialized with GBD data on LDL-C., SBP, IKF (CKD state) and FPG (DM state), as well as sex, age, location.  There are 3 ways a simulant is exposed to possible treatment:  background visits, post-MI visits (immediately after an MI), and follow-up visits (once they have been given Rx).  GBD healthcare utilization is used to determine how often a simulant goes to the doctor for background visits; MI incidence determines how often a simulant goes to the doctor for a "post MI" visit, and "follow-up" visits are determined by the assumption on follow-up frequency (3-6 months for BAU and 4-6 weeks for the 2 intervention scenarios).
For post-MI visits, the patient is given Rx with probability = 1.  LDL-C should be recorded in the simulation, but its value does not impact treatment decision in any of the 3 scenarios.
For background visits, the patient may or may not have their LDL-C measured, and the probability that they do measure LDL-C is given by the data in Table 1.
For follow-up visits, the LDL-C should be measured with probability = 1 and recorded in the simulation (to determine if the patient has reached target), and may impact Tx decisions (e.g. increasing dose if not at target).